Task ID 33452921 (#9580)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md

```powershell
reg add hklm\system\currentcontrolset\control\ci -v TestFlags -t REG_DWORD -d 0x300
```

## Appendix
A list of other relevant event IDs and their corresponding description.

> | Event ID | Description |
> |---|----------|

Reviewer: Cosmetic / codestyle: For future editors, it can be useful to point out the close relationship between the table title row and the table separation row (rows 1 & 2) in MarkDown tables by keeping the length of rows 1 & 2 matched:
```suggestion
| Event ID | Description |
| -------- | ----------- |
```
Please be aware that the lines of dashes can be used to align the text in both the title row and the values of the column, by using the colon `:` character as an indicator of  Left, Right, or Center alignment. (The main reason I haven't suggested much use of table alignment here in Windows IT-Pro Docs is that I am uncertain about how well it transfers to HTML in the docs.microsoft.com pages.)

| 3004 | Code Integrity could not verify the file as the page hash could not be found. |
| 3010 | The catalog containing the signature for the file under validation is invalid. |
| 3011 | Code Integrity finished loading the signature catalog. |
| 3012 | Code Integrity started loading the signature catalog. |
| 3023 | The driver file under validation did not meet the requirements to pass the application control policy. |
| 3024 | Windows application control was unable to refresh the boot catalog file. |
| 3026 | The catalog loaded is signed by a signing certificate that has been revoked by Microsoft and/or the certificate issuing authority. |
| 3033 | The file under validation did not meet the requirements to pass the application control policy. |
| 3034 | The file under validation would not meet the requirements to pass the application control policy if the policy was enforced. The file was allowed since the policy is in audit mode. | 
| 3036 | The signed file under validation is signed by a code signing certificate that has been revoked by Microsoft or the certificate issuing authority. |

> | 3064 | A user mode DLL under validation would not meet the requirements to pass the application control policy, if the policy was enforced. The DLL was allowed since the policy is in audit mode. | 

Reviewer 2: ```suggestion
| 3064 | A user mode DLL under validation would not meet the requirements to pass the application control policy if the policy was enforced. The DLL was allowed since the policy is in audit mode. | 
```

> | 3065 | [Ignored] A user mode DLL under validation would not meet the requirements to pass the application control policy, if the policy was enforced. | 

Reviewer 2: ```suggestion
| 3065 | [Ignored] A user mode DLL under validation would not meet the requirements to pass the application control policy if the policy was enforced. | 
```

| 3074 | Page hash failure while hypervisor-protected code integrity was enabled. |
| 3075 | This event monitors the performance of the Code Integrity policy check a file. |
| 3079 | The file under validation did not meet the requirements to pass the application control policy. |

> | 3080 | The file under validation would not have me the requirements to pass the application control policy, if the policy was in enforced mode. |

Reviewer 2: ```suggestion
| 3080 | The file under validation would not have met the requirements to pass the application control policy if the policy was in enforced mode. |
```

> | 3080 | If the policy was in enforced mode, the file under validation would not have meet the requirements to pass the application control policy. |

Reviewer 2: ```suggestion
| 3080 | If the policy was in enforced mode, the file under validation would not have met the requirements to pass the application control policy. |
```

| 3081 | The file under validation did not meet the requirements to pass the application control policy. |

> | 3082 | The non-WHQL driver would have been denied by the policy, if the policy was in enforced mode. | 

Reviewer 2: ```suggestion
| 3082 | The non-WHQL driver would have been denied by the policy if the policy was in enforced mode. | 
```

> | 3084 | Code Integrity will enforce theWHQL Required policy setting on this session. |

Reviewer 2: ```suggestion
| 3084 | Code Integrity will enforce the HQL Required policy setting on this session. |
```

> | 3085 | Code Integrity will not enforce theWHQL Required policy setting on this session. |

Reviewer 2: ```suggestion
| 3085 | Code Integrity will not enforce the HQL Required policy setting on this session. |
```
